meetings/2018-07-22-board-meeting-results.md

```diff
@@ -0,0 +1,59 @@
+# Ruby Together board meeting summary from July 22, 2018
+
+## Attendees and Agenda
+André, Coraline, Courteney, Valerie, Adarsh, Jonan were present.
+
+Topics for discussion included welcoming Jonan to the board, Work on the RubyMe OSS internship/mentorship proposal, review a new proposed granting process, and finally ratifying the new Executive Director role.
+
+## Welcome Jonan!
+
+The board is excited to welcome Jonan Scheffler as our newest board member.
+
+## Ruby Me
+The board reviewed and unanimously approved sponsorship of a new program called "RubyMe". It's an Open-Source internship which matches an experienced OSS developer with someone more junior (the Intern) to work on an OSS project together. From the proposal:
```

Comment:
> It's an Open-Source internship which matches an experienced OSS developer with someone more junior (the Intern) to work on an OSS project together.

We should probably update this to reflect what we have on RubyMe.org. (It's being messaged as an apprenticeship, not an internship.) @adarsh